domain/infosync/info.go

```diff
@@ -73,6 +80,8 @@ type InfoSyncer struct {
 	manager         util2.SessionManager
 	session         *concurrency.Session
 	topologySession *concurrency.Session
+	prometheusAddr  string
```

Comment:
need a lock to avoid the concurrent problems?

Reply:
It doesn't need a lock, the `prometheusAddr ` and `modifyTime ` are the field of `InfoSyncer`, which can be safely used by `globalInfoSyncer` which is a atomic.Value, and its `getGlobalInfoSyncer()` and `setGlobalInfoSyncer()`.